src/interface.ts

```diff
@@ -50,3 +50,4 @@ export * from './components/tab-panel/tab-panel.types';
 export * from './components/table/table.types';
 export * from './global/shared-types/separator.types';
 export * from './global/shared-types/icon.types';
+export * from './global/shared-types/img.types';
```

Comment:
also not sure if we need to export this

Reply:
Yes we do. Otherwise people can't use the type in the consuming code.